README.md

```diff
@@ -215,7 +215,7 @@ Start the daemon.
 
 `flags` - Flags array to be passed to the `ipfs daemon` command.
 
-`callback` is a function with the signature `function(err, ipfsApi)` that receives an instance of `ipfs-api` on success or an instance of `Error` on failure
+`callback` is a function with the signature `function(Error, ipfsApi)` that receives an instance of `Error` on failure or an instance of `ipfs-api` on success. 
```

Comment:
Why changing the  `err` name to `Error`? I think it is more correct to have `function  (err, ipfsApi)` and it will be also consistent with all other `js-ipfs-_` repos.

Reply:
Init method already uses Error and documentation.js detects the type and gives better output

Reply:
I see `documentation.js` detecting the type as an advantage in fact! Maybe we should create a separate discussion for this usage across all the projects, in order to understand the general impression. With this, we could make a consistent transition. What do you think @alanshaw  @hugomrdias ?